cli/ops/errors.rs

```diff
@@ -67,3 +72,13 @@ fn op_apply_source_map(
     "column": orig_column as u32,
   })))
 }
+
+fn op_format_diagnostic(
+  _state: &State,
+  args: Value,
+  _zero_copy: Option<ZeroCopyBuf>,
+) -> Result<JsonOp, ErrBox> {
+  let diagnostic = Diagnostic::from_json_value(&args);
+  let result = format!("{}", diagnostic.unwrap());
+  Ok(JsonOp::Sync(json!({ "result": result })))
```

Comment:
Can you use `?` instead of `unwrap()`?

Reply:
```
the trait `std::convert::From<std::option::NoneError>` is not implemented for `op_error::OpError`
```

So it requires implementing `NoneError`, which is still experimental, which I have done... not sure if you really want that though.

Reply:
Hmmm... can't use the feature on the stable channel, so I will have to remove it.

Reply:
Panicking is okay - I thought maybe `Diagnostic::from_json_value` returned a `Result`.. There's no useful information to be gained from `None`.